quickstart.md

```diff
@@ -161,253 +162,50 @@ NAME            HANDLER         AGE
-        "docker": {
-            "[REGISTRY_URL]": [
-                {
-                    "type": "sigstoreSigned",
-                    "keyPath": "kbs:///default/cosign-key/1"
-                }
-            ]
-        }
-    }
-}
-```
-
-Be sure to replace `[REGISTRY_URL]` with the desired registry URL of the encrypted image.
-
-Register the image pulling validation policy file to KBS storage:
-
-```shell
-mkdir -p $KBS_DIR_PATH/data/kbs-storage/default/security-policy
-cp security-policy.json $KBS_DIR_PATH/data/kbs-storage/default/security-policy/test
-```
-
-### Deploy encrypted image as a CoCo workload on CC HW
-
-Here is a sample yaml for encrypted image deploying:
-
-```shell
-cat << EOT | tee encrypted-image-test-busybox.yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  labels:
-    run: encrypted-image-test-busybox
-  name: encrypted-image-test-busybox
-  annotations:
-    io.containerd.cri.runtime-handler: [RUNTIME_CLASS]
-spec:
-  containers:
-  - image: [REGISTRY_URL]:encrypted
-    name: busybox
-  dnsPolicy: ClusterFirst
-  runtimeClassName: [RUNTIME_CLASS]
-EOT
-```
-
-Be sure to replace `[REGISTRY_URL]` with the desired registry URL of the encrypted image generated in previous step, replace `[RUNTIME_CLASS]` with kata runtime class for CC HW.
-
-Then configure `/opt/confidential-containers/share/defaults/kata-containers/configuration-<RUNTIME_CLASS_SUFFIX>.toml` to add `agent.aa_kbc_params=cc_kbc::<KBS_URI>` to kernel parameters. Here `RUNTIME_CLASS_SUFFIX` is something like `qemu-tdx`, `KBS_URI` is the address of Key Broker Service in KBS cluster like `http://123.123.123.123:8080`.
-
-Deploy encrypted image as a workload:
-
-```shell
-kubectl apply -f encrypted-image-test-busybox.yaml
-```
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may be required. See the [guides](./guides/) for more information.
+
+## Using CoCo
+
+Below is a brief summary and description of some of the CoCo use cases:
+
+- **Container Launch with Only Memory Encryption (No Attestation)** - Launch a container with memory encryption 
+- **Container Launch with Encrypted Image** - Launch an encrypted container by proving the workload is running 
+  in a TEE in order to retrieve the decryption key
+- **Container Launch with Image Signature Verification** - Launch a container and verify the authenticity and 
+  integrity of an image by proving the workload is running in a TEE
+- **Sealed secret** - Implement wrapped kubernetes secrets that are confidential to the workload owner and are
+  automatically decrypted by proving the workload is running in a TEE
+
+The CoCo use cases are implemented differently for each TEE, and they are described in each corresponding 
+[guide](./guides) section. To get started using CoCo without TEE hardware, follow the Non-TEE guide below:
+
+- [CoCo-dev](./guides/coco-dev.md)
+- [SEV(-ES)](./guides/sev.md)
+- [SNP](./guides/snp.md)
+- TDX
```

Comment:
If you are suggesting this path of abandoning the generic encrypted container yaml then are you creating a TDX guide?

Reply:
Intel folks are working on the TDX documentation that I will link here when done.

Reply:
Do you know who that might be?

Reply:
> Intel folks are working on the TDX documentation that I will link here when done.

we are not working on it. I'm not convinced we need TEE specific docs unless there's something very special about them that differs from `quickstart.md` flows.